addons/calendar/controllers/main.py

```diff
@@ -101,3 +101,20 @@ def notify(self):
     @http.route('/calendar/notify_ack', type='json', auth="user")
     def notify_ack(self):
         return request.env['res.partner'].sudo()._set_calendar_last_notif_ack()
+
+    @http.route('/calendar/join_videocall/<string:access_token>', type='http', auth='public')
+    def calendar_join_videocall(self, access_token):
+        event = request.env['calendar.event'].sudo().search([
+            ('access_token', '=', access_token)
+        ])
```

Comment:
I'm not sure how reliable psql is in terms of security, but we usually compare tokens with `consteq` and not directly with `search`. This would imply adding the event id in the route, which is probably a good thing to do anyway for easier tracking / debugging.

Reply:
@seb-odoo We also use `search` in `/calendar/meeting/join`. That's why I implemented it like this. From what I have already seen in the codebase, we use `consteq` a lot in portal routes, but not sure about user authenticated routes. If we use consteq here, it would probably make sense to change the `/calendar/meeting/join` route as well